decimalSuite_test.go

```diff
@@ -120,14 +147,19 @@ func runTest(testVals decValContainer, testValStrings testCaseStrings) error {
 			testValStrings.testFunc,
 			testValStrings.val2,
 			calcRestul,
-			testVals.expected)
+			testValStrings.expectedResult)
 	}
 	return nil
 }
 
 // TODO: get runTest to run more functions such as FMA.
 // execOp returns the calculated answer to the operation as Decimal64.
 func execOp(val1, val2 Decimal64, op string) Decimal64 {
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println("Recovered from", r)
+		}
+	}()
```

Comment:
Why is this needed?

Reply:
to stop the testing function from crashing when a Nan is signalled (from signalNaN64())

Reply:
removed